src/components/KLabel/KLabel.vue

```diff
@@ -87,5 +105,9 @@ const hasTooltip = computed((): boolean => !!(props.info || props.help || slots.
       }
     }
   }
+
+  .kong-icon {
+    margin-left: var(--spacing-xxs, var(--kui-space-20, $kui-space-20));
+  }
```

Comment:
does this need the `:deep` selector?

Reply:
I would think so but it works without it so I left it as is.